src/Functions/FunctionBase64Conversion.h

```diff
@@ -22,9 +22,19 @@ namespace ErrorCodes
     extern const int INCORRECT_DATA;
 }
 
+enum class Base64Variant : uint8_t
+{
+    Normal,
+    Url
+};
+
+extern std::vector<UInt8> preprocessBase64Url(const std::span<const UInt8> src);
```

Comment:
Let's remove `extern` from l. 31 and 32.

Also, my local clang tidy complains that the `const` qualifier for `src` is superfluous since the `std::span<const UInt8>` is passed by value. Interestingly, it is passed in the same way in l. 66 and 101 and clang tidy did not complain. I guess that was because the other places are templatized. In any case, it would be nice if you could remove `const` here and in other places where `std::span` is passed by value.